modules/custom/openy_group_schedules/templates/groupex-all-search-results.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
+/**
+ * @file
+ * Groupex search results template.
+ *
+ * Available variables:
+ * - form: search form;
+ * - schedule: (string) search results;
+ * - empty_results: bool flag if there any results found;
+ *
+ * @ingroup themeable
+ */
+#}
+<section id="component_160855" class="{{ interval }}-results accessible-buttons" type="ymca_twin_cities_group_ex_search_results">
```

Comment:
In case if component_160855 name doesn't exist in css - let's remove it and make less legacy ))))